tests/testsuite/fix.rs

```diff
@@ -0,0 +1,3066 @@
+//! Tests for the `cargo fix` command.
+//! https://github.com/rust-lang/cargo/blob/f1bf94d3b2a779d14e2ad6dff43f8d8017583b0d/tests/testsuite/fix.rs
+
+//use cargo::core::Edition;
+use cargo_test_support::compare::assert_e2e;
+use cargo_test_support::git::{self, init};
+use cargo_test_support::paths;
+use cargo_test_support::prelude::*;
+use cargo_test_support::registry::{Dependency, Package};
+use cargo_test_support::str;
+use cargo_test_support::tools;
+use cargo_test_support::{basic_manifest, is_nightly, project};
+
+#[cargo_test]
+fn do_not_fix_broken_builds() {
+    let p = project()
+        .file(
+            "src/lib.rs",
+            r#"
+                pub fn foo() {
+                    let mut x = 3;
+                    let _ = x;
+                }
+
+                pub fn foo2() {
+                    let _x: u32 = "a";
+                }
+            "#,
+        )
+        .build();
+
+    p.cargo("fix --allow-no-vcs")
```

Comment:
`p.cargo` won't work .

Changing all of these to
https://github.com/crate-ci/cargo-fixit/blob/2c930c66d154f87a0902866988cdfdf170f4461f/tests/testsuite/help/mod.rs#L5C6-L5C58
 would be annoying

An extension trait on `Project` could help, kind of like
https://github.com/crate-ci/cargo-release/blob/9c5c833a2e61ee5224e7f5347abf3a914dd0e9ac/tests/testsuite/main.rs#L36-L52